tests/declare_export/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -204,7 +202,6 @@ declare export default class Foo { givesANum(): number; };
  */
 
 declare export default class Foo { givesANum(): number }
-
```

Comment:
This line was not (this is the end of the file, there should be no spurious empty lines)